odoo/addons/base/models/ir_model.py

```diff
@@ -1267,8 +1267,8 @@ def _reflect_selections(self, model_names):
         for field in fields:
             model = self.env[field.model_name]
             for value, modules in field._selection_modules(model).items():
-                if module in modules:
-                    xml_id = selection_xmlid(module, field.model_name, field.name, value)
+                for m in modules:
+                    xml_id = selection_xmlid(m, field.model_name, field.name, value)
```

Comment:
Please put back the variable `module`.

Reply:
that variable name bothered me too but using `for module in modules` would erase the `module` argument variable, we could change it to something else but that variable is just used at one place and on the following line so I think it is ok we leave the variable as it is